plugin/core/completion.py

```diff
@@ -55,7 +55,11 @@ def format_completion(item: dict, word_col: int, settings: 'Settings') -> 'Tuple
             trigger = trigger[1:]  # remove clangd insertion indicator
         else:
             debug("replacement prefix does not match trigger!")
-            replacement = item.get("insertText") or trigger
+            if trigger == "Implement all members":
```

Comment:
Let's remove the server-specific logic, remove the assignments to ` replacement`, and update the warning to:
```python
debug("WARNING: Replacement prefix does not match trigger '{}'".format(trigger))
```